### PR TITLE
"Iframe element has non-empty accessible name" [cae760]: use `tabindex`

### LINKED ISSUE #2015: "Iframe element has non-empty accessible name" [cae760]: use `tabindex`?
In [Iframe element has non-empty accessible name](https://www.w3.org/WAI/standards-guidelines/act/rules/cae760/proposed/), we target "(`iframe`) that can be accessed by [sequential focus navigation](https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation)."

Based on what I remember from the lengthy discussion in #1855 about [Iframe with interactive elements is not excluded from tab-order](https://www.w3.org/WAI/standards-guidelines/act/rules/akn7bn/proposed/), I think we should change that to "that does not have a negative `tabindex`".
That would at least align the vocabulary used, and in the former we did discuss a lot about the wording before deciding on using the `tabindex`.

### PR DESCRIPTION
**Pull request by Jym77**
Replace "can be accessed by sequential focus navigation" with "non-negative tabindex".
This align with the recent changes made in #1855 on the other rule with iframes and tabindexes.

Closes issue(s):

- closes #2015

Need for Call for Review:
This will require a 1 week Call for Review (aligning phrasing without changing intend)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [X] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/iframe-non-empty-accessible-name-cae760.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to `iframe` elements that are [included in the accessibility tree][] and that can be accessed by [sequential focus navigation][].
+This rule applies to `iframe` elements that are [included in the accessibility tree][] and that do not have an negative `tabindex` [attribute value][].
```
Comment (giacomo-petri) [inline _rules/iframe-non-empty-accessible-name-cae760.md]:
```suggestion
This rule applies to `iframe` elements that are [included in the accessibility tree][] and that do not have a negative `tabindex` [attribute value][].
```

File: _rules/iframe-non-empty-accessible-name-cae760.md [hunk 2]
```diff
@@ -136,15 +136,16 @@ This `iframe` is not [included in the accessibility tree][] because of setting a
...
-This `iframe` element has a negative `tabindex` and therefore is not included in the [sequential focus navigation][].
+This `iframe` element has a negative `tabindex` [attribute value][].
 
 ```html
 <iframe tabindex="-1" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
 ```
 
 [accessible name]: #accessible-name 'Definition of accessible name'
+[accessible name and description computation]: https://www.w3.org/TR/accname
+[attribute valiue]: #attribute-value 'Definition of Attribute value'
```
Comment (giacomo-petri) [inline _rules/iframe-non-empty-accessible-name-cae760.md]:
```suggestion
[attribute value]: #attribute-value 'Definition of Attribute value'
```

**Review by giacomo-petri — CHANGES_REQUESTED** [review]

**Review by HelenBurge — APPROVED** [review]

**Review by carlosapaduarte — APPROVED** [review]

File: _rules/iframe-non-empty-accessible-name-cae760.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to `iframe` elements that are [included in the accessibility tree][] and that can be accessed by [sequential focus navigation][].
+This rule applies to `iframe` elements that are [included in the accessibility tree][] and that do not have a negative `tabindex` [attribute value][].
```
Comment (ajanec01) [inline _rules/iframe-non-empty-accessible-name-cae760.md]:
```suggestion
This rule applies to `iframe` elements that are [included in the accessibility tree][] and do not have a negative `tabindex` [attribute value][].
```
Maybe its me but it I find that to sound a bit better.

**Review by giacomo-petri — APPROVED** [review]

**Review by ajanec01 — APPROVED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Comment by Jym77** [comment]
Call for review ends on February 16th.

**Review by catdro — APPROVED** [review]

**Comment by Jym77** [comment]
Call for review has ended. Merging.

**Pull request merged**